### ISSUE TITLE
Partners should not be able to modify client epxiration fields.

### ISSUE THREAD

**Issue by akoebbe** [happybottoms/coverd#134]
Partners can view expiration fields, but should not be able to edit those fields on clients.

![image](https://user-images.githubusercontent.com/1044465/86877410-75b62280-c0ac-11ea-8968-a69ff103de92.png)

**Comment by scadergit** [comment]
Does that include the override? Should that even show if user is a partner role?

**Comment by akoebbe** [comment]
Yeah. It's ok for the partner to see that it's overridden. That will tip them off that the bank has made a change. Normally the bank will get a call from the partner saying "Hey there's this one client we need to make an exception for, is that cool with you?". This give the bank full control and avoids abuse. If the exception is granted the Partner will see that checkbox to know that the bank has made the change.

Make sense?

**Comment by akoebbe** [comment]
@scadergit ^^

(Sorry failed to mention you)
